stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -2472,11 +2471,16 @@ internal constructor(
                 )
             }
 
```

Comment:
Don't want to bother with self-signed certificates. That's why using unencrypted protocols here.